packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,864 @@
+
+#### Request Query Parameters
+
+None
+
+#### Response Body
+
+A [Rule Object](#okta-identity-engine-policy-and-rule-objects).
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/${policyId}/rules/${ruleId}"
+```
+
+##### Response
+
+```json
+{
+    "name": "Catch-all Rule",
+    "id": "rul10y0LatXn0HP2p0g4",
+    "type": "Okta:IdpRouting",
+    "priority": 0,
+    "conditions": [],
+    "action": "ALLOW",
+    "requirement": {
+        "idpId": "OKTA",
+        "type": "okta_idp"
+    },
+    "status": "ACTIVE",
+    "default": false,
+    "_links": {
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4/rules/rul10y0LatXn0HP2p0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4/rules/rul10y0LatXn0HP2p0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "policy": {
+            "href": "https://${yourOktaDomain}/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Syntax for All Other Operations
+
+See <https://developer.okta.com/docs/reference/api/policy/#policy-api-operations> for request syntax for all other operations.
+
+## Okta Identity Engine Policy and Rule Objects
+
+Okta Identity Engine introduces a set of new Policy and Rule objects that are different from the existing Policy and API objects:
+
+* [IdP Routing Policy Object](#idp-routing-policy-object)
+* [IdP Routing Rule Object](#idp-routing-rule-object)
+* [Identifier Match Policy Object](#identifier-match-policy-object)
+* [Identifier Match Rule Object](#identifier-match-rule-object)
+* [Unknown User Policy Object](#uknown-user-policy-object)
+* [Unknown User Rule Object](#unknown-user-rule-object)
+* [Sign On Policy Object](#sign-on-policy-object)
+* [Sign On Rule Object](#sign-on-rule-object)
+* [User Profile Policy Object](#user-profile-policy-object)
+* [User Profile Rule Object](#user-profile-rule-object)
+
+Default instances of each Policy type are created automatically when Okta Identity Engine is enabled for your org. Each of the default Policy objects also has a default Rule object that is created automatically.
+
+The `conditions` objects, which can be specified in Rule objects, are documented in the existing Policy API documentation at: <https://developer.okta.com/docs/reference/api/policy/#conditions>
```

Comment:
This is not right, the conditions are different from the existing ones we have. I would take out this entire sentence.

Reply:
Removing sentence. For now, examples will be only documentation of conditions.